assets/src/domain/blocks/event-attendees/controls/AttendeeLimit.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { TextControl } from '@wordpress/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+import { AttendeesEditProps } from '../types';
+
+const AttendeeLimit: React.FC<AttendeesEditProps> = ({ attributes, setAttributes }) => {
+	const { limit } = attributes;
+
+	// TODO get value
+	const attendeesCount = 0;
```

Comment:
does this need updating as part of this PR?

Reply:
No, it will be in Data PR